### PR TITLE
show all namespaces by default

### PR DESCRIPTION
**Pull request by rade**
There is nothing special about the "default" namespace, and selecting it by default means we only show a partial view of the system, which is hard to spot, especially for novice users.

### CODE REVIEW

**Comment by rade** [comment]
I have tested this against a report from dev, so with some k8s in it, but not docker swarm. If somebody could check that please, that would be great.

File: app/api_topologies.go [hunk 1]
```diff
@@ -50,13 +50,9 @@ var (
 )
 
 // namespaceFilters generates a namespace selector option group based on the given namespaces
-func namespaceFilters(namespaces []string, defaultNamespace, noneLabel string) APITopologyOptionGroup {
-	options := APITopologyOptionGroup{ID: "namespace", Default: "", SelectType: "union", NoneLabel: noneLabel}
+func namespaceFilters(namespaces []string, noneLabel string) APITopologyOptionGroup {
+	options := APITopologyOptionGroup{ID: "namespace", Default: "none", SelectType: "union", NoneLabel: noneLabel}
```
Comment (rade) [inline app/api_topologies.go]: The `"none"` here took me a while to figure out; it is a dependency on a constant in the UI code.

I initially had `""` and then `noneLabel`, neither of which worked.

Comment (ekimekim) [inline app/api_topologies.go]: That is a bug in the UI code then. `""` should be correct.

Comment (ekimekim) [inline app/api_topologies.go]: This means that the UI is currently bugged in cases where there's no default namespace?

**Pull request merged**